mindsdb/integrations/handlers/google_fit_handler/google_fit_tables.py

```diff
@@ -0,0 +1,71 @@
+from mindsdb.integrations.libs.api_handler import APITable
+from mindsdb.integrations.libs.response import HandlerResponse as Response
+from mindsdb.integrations.utilities.sql_utils import extract_comparison_conditions
+from mindsdb_sql.parser import ast
+import datetime
+import pytz
+import time
+from tzlocal import get_localzone
+
+class GoogleFitTable(APITable):
+
+    def time_parser(self, args) -> int:
+        ymd = args.split('-')
+        epoch0 = datetime.datetime(1970, 1, 1, tzinfo=pytz.utc)
+        time = pytz.timezone(str(get_localzone())).localize(datetime.datetime(int(ymd[0].rstrip()), int(ymd[1].rstrip()), int(ymd[2].rstrip())))
```

Comment:
Can you add docstring to this method so users can know what this method does? Also can we simplify this?

Reply:
Just done the comment.